src/main/java/no/nav/veilarboppfolging/client/veilarbarena/VeilarbarenaClientImpl.java

```diff
@@ -76,10 +81,13 @@ public Optional<VeilarbArenaOppfolging> hentOppfolgingsbruker(Fnr fnr) {
     @SneakyThrows
     @Override
     public Optional<ArenaOppfolging> getArenaOppfolgingsstatus(Fnr fnr) {
+		PersonRequest personRequest = new PersonRequest();
+		personRequest.setFnr(fnr);
         Request request = new Request.Builder()
-                .url(joinPaths(veilarbarenaUrl, "/api/oppfolgingsstatus/" + fnr.get()))
+                .url(joinPaths(veilarbarenaUrl, "/api/v2/oppfolgingsstatus/"))
```

Comment:
Fjerne trailing-slash:
```suggestion
                .url(joinPaths(veilarbarenaUrl, "/api/v2/oppfolgingsstatus"))
```